includes/admin/class-sensei-setup-wizard.php

```diff
@@ -586,18 +589,47 @@ function( $extension ) use ( $installing_plugins, $selected_plugins ) {
 	 * @param string[] $extension_slugs Extension slugs to install.
 	 */
 	public function install_extensions( $extension_slugs ) {
-		$extensions_to_install = array_filter(
-			array_map(
-				function( $extension ) use ( $extension_slugs ) {
-					$key       = array_search( $extension->product_slug, $extension_slugs, true );
-					$is_wc_com = isset( $extension->wccom_product_id );
-
-					return ( false !== $key && ! $is_wc_com ) ? $extension : false;
-				},
-				$this->get_sensei_extensions()
-			)
-		);
+
+		$extensions_to_install = [];
+		$wccom_extensions      = [];
+
+		foreach ( $this->get_sensei_extensions() as $extension ) {
+			if ( ! in_array( $extension->product_slug, $extension_slugs, true ) ) {
+				continue;
+			}
+
+			if ( isset( $extension->wccom_product_id ) ) {
+				$wccom_extensions[] = $extension->plugin_file;
+			} else {
+				$extensions_to_install[] = $extension;
+			}
+		}
+
+		set_transient( self::WCCOM_INSTALLING, $wccom_extensions, 10 * 60 );
```

Comment:
I liked this approach because if it fails installing automatically, we also can track if they install manually.
And because of that, I'm just wondering if it would worth adding a whole day instead of 10 minutes. WDYT?

Reply:
I'm not sure that would really track the success of the setup wizard install, and that we want to register the manual install as part of it. With the short time, we can infer the % that likely succeeded with the auto-install. So maybe a separate event would be better for the manual one?

Reply:
Hum, maybe a separate event for that could be a good idea! In this case, I think we could leave it for the future, if we really think this information is important. :)